src/pydata_sphinx_theme/__init__.py

```diff
@@ -1084,6 +1085,43 @@ def setup_translators(app):
             app.set_translator(name, translator, override=True)
 
 
+# ------------------------------------------------------------------------------
+# customize method for logo management
+# ------------------------------------------------------------------------------
+
+
+def setup_logo_path(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: nodes.Node
+) -> None:
+    """Set up relative paths to logos independant from the version"""
+
+    # get informations from the context
+    pathto = context.get("pathto")
+    logo = context.get("logo")
+    image_light = context.get("theme_logo", {}).get("image_light")
+    image_dark = context.get("theme_logo", {}).get("image_dark")
```

Comment:
I don't understand this. If `context` doesn't have `"theme_logo"` then you are effectively calling `{}.get('image_light')` which will always fail?

Reply:
yes, but that was saving me from a "`if` statement". When I tested it, nothing failed, you just get `image_light = None` which is then resoved downstream

Reply:
This seems reasonable to me - maybe we can just add a comment like "If the keyword isn't defined this will return None and be resolved later on"

Reply:
Am I missing something here? Does calling `.get()` on an empty dict return None? For non empty dict it will raise a KeyError, unless you provide a fallback

Reply:
> Does calling `.get()` on an empty dict return `None`

Yes, I think that's the reason that @12rambau is doing this. The implicit fallback is `None`.

Reply:
`.pop` return an error but `.get` return `None` which is exactly what I'm using

Reply:
argh, I should really stop doing GitHub on my phone before I've had caffeine 🤦🏻 I forgot that None was the default fallback for `get()`.